hdat/store.py

```diff
@@ -50,20 +55,37 @@ def select_recent(self, i, *args):
         else:
             return self.read_result(os.path.join(top_directory, recent_id))
 
-    def select_recents_suite(self, *args):
```

Comment:
@russellkan it looks like the rest of the code here is incomplete; I will hold off reviewing for now.